File: scraper/comment.py
class comment(object):
    def __init__(self, author, body, body_html, created_utc, distinguished, edited, cid,
                 is_submitter, link_id, parent_id, permalink, saved, score,
                 stickied, submission, subreddit, subreddit_id) -> None:
        self.type = "comment"
        self.author = author
        self.body = body
        self.body_html = body_html
        self.created_utc = created_utc
        self.distinguished = distinguished
        self.edited = edited
        self.id = cid
        self.is_submitter = is_submitter
        self.link_id = link_id
        self.parent_id = parent_id
        self.permalink = permalink
        self.comment_link = "https://reddit.com" + permalink
        self.saved = saved
        self.score = score
        self.stickied = stickied
        self.submission = submission
        self.subreddit = "r/" + subreddit
        self.subreddit_id = subreddit_id

File: graphql/src/models/comment.js
const mongoose = require("mongoose");
const Schema = mongoose.Schema;

const commentSchema = new Schema({
  type: {
    type: String,
    required: true,
  },
  author: {
    type: String,
    required: true,
  },
  clicked: {
    type: Boolean,
    required: true,
  },
  created_utc: {
    type: Number,
    required: true,
  },
  distinguished: {
    type: String,
    required: false,
    default: null,
  },
  edited: {
    type: Boolean,
    required: true,
  },
  reddit_id: {
    type: String,
    required: true,
  },
  is_submitter: {
    type: Boolean,
    required: true,
  },
  link_id: {
    type: String,
    required: true,
  },
  parent_id: {
    type: String,
    required: true,
  },
  permalink: {
    type: String,
    required: true,
  },
  comment_link: {
    type: String,
    required: true,
  },
  saved: {
    type: Boolean,
    required: true,
  },
  score: {
    type: Number,
    required: true,
  },
  stickied: {
    type: Boolean,
    required: true,
  },
  subreddit: {
    type: String,
    required: true,
  },
  subreddit_id: {
    type: String,
    required: true,
  },
});

module.exports = mongoose.model("Comment", commentSchema);
